src/box/tuple.h

```diff
@@ -375,6 +384,18 @@ tuple_check_data_offset(uint32_t data_offset)
 	return 0;
 }
 
+/**
+ * The that the tuple will be compact if initialized.
+ * @param data_offset - see member description in struct tuple.
+ * @param bsize - see member description in struct tuple.
+ * @return
+ */
+static inline bool
+tuple_can_be_compact(uint16_t data_offset, uint32_t bsize)
+{
+	return data_offset < INT8_MAX && bsize < UINT8_MAX;
```

Comment:
Why '<', not '<='? Why INT8_MAX, not UINT8_MAX for data_offset? Please add a comment explaining this.

Reply:
The max value that can be stored in 7/8 bits is INT8_MAX/UINT8_MAX. So why <, not <=?

Reply:
Thanks, that was a mistake!

Reply:
fixed